annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -39,6 +39,7 @@
      * @throws PropertyValidationException
      */
     protected void validateAgentBody(Body body) throws ParamValidationException, PropertyValidationException {
+	if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown